README.md

```diff
@@ -2,6 +2,102 @@
 
 A migration tool does migrations for Konflux pipelines.
 
+Doing migrations is the major feature of pipeline-migration-tool. It auto-discovers migrations for
+given task bundle upgrades and applies found migrations to build pipelines. The pipeline can be
+either included in a `PipelineRun` definition as `spec.pipelineSpec` or a single `Pipeline` defintion.
```

Comment:
We need to say that it runs as a post upgrade task of Renovate.

Reply:
Updated.